101-traffic-manager-virtual-machines/azuredeploy.json

```diff
@@ -0,0 +1,271 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
```

Comment:
I see.  From a traffic manager perspective, the external endpoints example is a more complex example than either the VM or WebApp one because user needs to specify more cryptic parameters such as the location of the nearest Azure dc.  Maybe they should all be 201s.

From: Kay Singh [mailto:notifications@github.com]
Sent: 04 November 2015 19:05
To: Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Gareth Bradshaw <Gareth.Bradshaw@microsoft.com>
Subject: Re: [azure-quickstart-templates] Azure Traffic Manager ARM examples (#796)


In 101-traffic-manager-virtual-machines/azuredeploy.json<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-quickstart-templates%2fpull%2f796%23discussion_r43923358&data=01%7c01%7cgarbrad%40064d.mgd.microsoft.com%7ca6169a2cd2b54865174108d2e54ad8ae%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=OdhAjOlslY5Ro4CBVvBREgjt8xFJ5%2bSKTARSIkkzWtc%3d>:

> @@ -0,0 +1,271 @@

> +{

> +    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",

True but think about it from the perspective of the user who is starting to learn about IaaS. Asking them to learn about complex VM scenarios in 101 seems a bit much :) Which is why we usually reserve more complex VM scenarios for 201

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-quickstart-templates%2fpull%2f796%2ffiles%23r43923358&data=01%7c01%7cgarbrad%40064d.mgd.microsoft.com%7ca6169a2cd2b54865174108d2e54ad8ae%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=SNgfnXZsBzOWawL6Y1VSKVl45NWvSoDzUotEywaPRIs%3d>.